src/js/users/components/Permission.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import styled from "styled-components";
+import { BoxGroupSection, Icon } from "../../base";
+
+const StyledPermissionIcon = styled(Icon)`
+    margin-right: 15px;
+`;
+
+const StyledPermissionItem = styled(BoxGroupSection)`
+    align-items: center;
+    display: flex;
+`;
+
+/**
+ * Permission item with an icon indicating its status.
+ * @prop permission - name of the permission.
+ * @prop value - indicates if the permission is granted.
+ */
+export function PermissionItem({ permission, value }: { permission: string; value: boolean }) {
```

Comment:
To keep it more consistent with the other components, define a type  `PermissionItemProps` with the permission item types and doc strings.